superset/views/base_api.py

```diff
@@ -180,8 +180,19 @@ class BaseTagFilter(BaseFilter):  # pylint: disable=too-few-public-methods
     """ The Tag class_name to user """
     model: type[Dashboard | Slice | SqllabQuery | SqlaTable] = Dashboard
     """ The SQLAlchemy model """
+    id_based_filter = False
 
     def apply(self, query: Query, value: Any) -> Query:
+        # ID based filter
+        if self.id_based_filter:
```

Comment:
Can we write a small unit test to verify the query is properly grabbing the write rows?

Reply:
Hey @hughhhh -- just added unit tests for the `BaseTagFilter` -- the `ChartTagNameFilter`, `ChartTagIdFilter`, `DashboardTagNameFilter`, `DashboardTagIdFilter`, `SavedQueryTagNameFilter` and `SavedQueryTagIdFilter` should have integration tests already 🙌 

Let me know if you have any other concerns or feedback 🙏